crates/.cargo/config.toml

```diff
@@ -1,4 +1,5 @@
 [target.thumbv7em-none-eabihf]
+runner = "../tools/cargo-runner.sh"
```

Comment:
This works reliably from inside the `crates/` directory, but I have not tested from elsewhere yet.

Reply:
It turns out that `target.<cfg>.runner` also works, so I might be able to specify a Windows script here as well with some effort.

Reply:
I can't figure out an easy way to get this to work, so given that Unix platforms almost always have Python, and you're clearly already requiring it on Windows, let's just use that for now.